app/routes/admin/tenants.py

```diff
@@ -256,14 +387,52 @@ async def get_wallet_auth_token(
     return response
 
 
-@router.put("/{wallet_id}", response_model=Tenant)
+@router.put("/{wallet_id}", response_model=Tenant, summary="Update Tenant by Wallet ID")
 async def update_tenant(
     wallet_id: str,
     body: UpdateTenantRequest,
     group_id: Optional[str] = group_id_query,
     admin_auth: AcaPyAuthVerified = Depends(acapy_auth_tenant_admin),
 ) -> Tenant:
-    """Update tenant by id."""
+    """
+    Update Tenant by Wallet ID
+    ---
+
+    Update a Tenant's details based on their Wallet ID.
+
+    Holders cannot have their roles updated. Attempting to assign issuer or verifier
+    roles to a holder will result in a 409 conflict error.
+
+    Only issuers or verifiers can be updated to hold both roles.
+
+    This endpoint does not support revoking roles from issuers or verifiers.
+
+    Updates to `image_url` for issuers and verifiers will be reflected on the trust registry.
```

Comment:
NB: Does updating wallet_label get reflected on the trust registry? I haven't checked, but noting down as important question

Reply:
Ok, it does. ✅